dev/Code/Framework/AzFramework/AzFramework/Application/Application.cpp

```diff
@@ -317,11 +311,6 @@ namespace AzFramework
         {
             m_commandLine.Parse(*m_argC, *m_argV);
         }
-        else
-        {
-            m_commandLine.Parse();
```

Comment:
in current implementation m_argC and m_argV is always initialized (line 162 - line 172), so this code create a false assumption.

Reply:
Not sure I really follow how the arguments are always initialized? I can see the changes made to the Launchers might be useful but I don't see the need for the Application changes. Do you agree?